### DISCUSSION TITLE
Hide upstream/downstream links in asset graph

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17875] [Q&A]
QQ: when selecting assets in the dagit UI, how do you hide the typically hints at "upstream/downstream" nodes? This is to take screenshots for powerpoints, etc...

E.g. here the upstream are hinted at (`raw_orders`, `raw_payments`, etc..)

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699553795753309?thread_ts=1699553795.753309&cid=C01U954MEER).*

**Comment by bengotow** [comment]
Hey there! This is a good question. I'm not sure we have a good solution now, but I think there are two things that might help:

- If you go to the Asset Details > Lineage page for one of your assets, you can capture a slice of your graph that is more explicitly bounded to "assets within X steps of this asset". That view clips more aggressively to the specified depth and won't show the dangling "out of scope" links.

- If you're comfortable editing an SVG, you can click the "Download SVG" button in the bottom right of the asset graph DAG view and cut the links out. At least with our current styling, this regexp will match the links in the SVG. If you open the SVG as a text document, do a regexp find / replace with "", this should do it! 

```<foreignObject ([^>]*) height="90".*?</foreignObject>```

**Comment by yamhirotoCS** [comment]
@bengotow any chance to have a toggle button on the UI to mask those  _dangling "out of scope" links_ ?  When navigating an asset graph of a big job, those nodes get distracting.
